release/build.sh

```diff
@@ -175,16 +171,15 @@ elif [ -z "$all" ]; then
 		exit 1
 	fi
 	echo "Building $BRNAME for $os/$arch"
-	GOOS="$os" GOARCH="$arch" $GODEP $GOBIN build -o "${dir}/${BRNAME}${label}${timestamp}.$os.$arch"
+	GOOS="$os" GOARCH="$arch" $GODEP $GOBIN build -ldflags "-s" -o "${dir}/${BRNAME}${label}${timestamp}.$os.$arch" $GOPACKAGE
 else
-	echo "Building $BRNAME for common OS/ARCH:"
+	echo "Building $BRNAME for all OS/ARCH:"
 	os="$OSVALUES"
 	arch="$ARCHVALUES"
 	for j in $arch; do
-		printf "  "
 		for i in $os; do
-			printf "$i/$j "
-			GOOS="$i" GOARCH="$j" $GODEP $GOBIN build -o "${dir}/${BRNAME}${label}${timestamp}.$i.$j"
+			printf "    $i/$j \n"
```

Comment:
Rather than using `printf "...\n"` it would be better to just use `echo "..."`; then remove line 184.